hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/MockNodes.java

```diff
@@ -165,6 +167,11 @@ public String getHostName() {
       return this.hostName;
     }
 
+    @Override
+    public String getClusterID() {
```

Comment:
This needs some tests.

Reply:
I will add a test in TestNodesListManager